bigquery/snippets/snippet.go

```diff
@@ -1468,6 +1587,57 @@ func importCSVExplicitSchema(client *bigquery.Client, datasetID, tableID string)
 	return nil
 }
 
+func importCSVAutodetectSchema(client *bigquery.Client, datasetID, tableID string) error {
+	ctx := context.Background()
+	// [START bigquery_load_table_gcs_csv_autodetect]
+	gcsRef := bigquery.NewGCSReference("gs://cloud-samples-data/bigquery/us-states/us-states.csv")
+	gcsRef.SourceFormat = bigquery.CSV
+	gcsRef.AutoDetect = true
+	gcsRef.SkipLeadingRows = 1
+	loader := client.Dataset(datasetID).Table(tableID).LoaderFrom(gcsRef)
+
+	job, err := loader.Run(ctx)
+	if err != nil {
+		return err
+	}
+	status, err := job.Wait(ctx)
+	if err != nil {
+		return err
+	}
+
+	if status.Err() != nil {
+		return fmt.Errorf("Job completed with error: %v", status.Err())
```

Comment:
Nit: errors should start with a lower case letter. https://github.com/golang/go/wiki/CodeReviewComments#error-strings